keps/sig-scheduling/3094-pod-topology-spread-considering-taints/README.md

```diff
@@ -318,27 +322,80 @@ not change the default behavior.
 
 All code is expected to have adequate tests (eventually with coverage
 expectations). Please adhere to the [Kubernetes testing guidelines][testing-guidelines]
 when drafting this test plan.
 
 [testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
 -->
-- Unit and integration tests:
-  - Defaulting and validation tests
-  - Feature gate enable/disable tests
-  - `NodeAffinityPolicy` and  `NodeTaintsPolicy` work as expected
-- Benchmark tests:
-  - Verify the performance of looping all toleration and taints in calculating skew is acceptable
+
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+<!--
+Based on reviewers feedback describe what additional tests need to be added prior
+implementing this enhancement to ensure the enhancements have also solid foundations.
+-->
+No.
+
+##### Unit tests
+
+<!--
+In principle every added code should have complete unit test coverage, so providing
+the exact set of tests will not bring additional value.
+However, if complete unit test coverage is not possible, explain the reason of it
+together with explanation why this is acceptable.
+-->
+
+<!--
+Additionally, for Alpha try to enumerate the core package you will be touching
+to implement this enhancement and provide the current unit coverage for those
+in the form of:
+- <package>: <date> - <current test coverage>
+The data can be easily read from:
+https://testgrid.k8s.io/sig-testing-canaries#ci-kubernetes-coverage-unit
+
+This can inform certain test coverage improvements that we want to do before
+extending the production code to implement this enhancement.
+-->
+
+- `pkg/api/pod`: `2022-06-17` - `66.7%`
+- `pkg/apis/core/validation`: `2022-06-17` - `82.1%`
+- `pkg/scheduler`: `2022-06-17` - `75%`
+- `pkg/scheduler/framework/plugins/defaultpreemption`: `2022-06-17` - `85.2%`
+- `pkg/scheduler/framework/plugins/podtopologyspread`: `2022-06-17` - `86%`
+
+##### Integration tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+-->
+
+- tests with policies honored/ignored in filtering
+- tests with policies honored/ignored in scoring
+
+##### e2e tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+
+We expect no non-infra related flakes in the last month as a GA graduation criteria.
+-->
+None.
```

Comment:
explain why.

For example, you can say that we are not introducing new API endpoints and that the feature is independent from kubelet, so integration test coverage will be enough.

Reply:
Thanks for the advices, updated.